components/roster/heroes/HeroPortrait.tsx

```diff
@@ -0,0 +1,118 @@
+"use client";
+
+import Image, { StaticImageData } from "next/image";
+import { Hero } from "@/lib/roster";
+import { AscensionLevel, CharacterClass, Faction } from "@/lib/characters";
+import { heroImages, rosterImages, tekImages } from "@/lib/images";
+
+type PortraitProps = {
+  hero: Hero;
+};
+
+function getBackground(ascensionLevel: AscensionLevel): StaticImageData {
```

Comment:
just make an object instead of using a switch statement.

```ts
ascensionLevelBackgrounds = {
  AscensionLevel.Elite: rosterImages.eliteBackground,
  ...
}
```

then you can just do a normal object lookup instead.

Reply:
Not sure I follow how to do this.. It doesn't like this code. I was looking into another way of doing it like mapping the enum values to an object, but it just created 
```
{
    "Elite": undefined,
    "Elite+": undefined,
    ...    
}
```

Reply:
Fixed, lmk if this makes sense!